.env.example

```diff
@@ -10,6 +10,22 @@ CUBEJS_DB_TYPE=postgres
 CUBEJS_API_SECRET= # random 32-digit base64 string - generate from https://generate.plus/en/base64
 CUBEJS_DEV_MODE=true
 
+PORT=4000
+
 # local web server
-APP_API_PORT=4000
```

Comment:
Why this change?

Reply:
there is a default cubejs variable `PORT` that is used by cubejs to switch the port. I think we can use the same for docker-compose as well.